static/app/views/performance/data.tsx

```diff
@@ -649,7 +649,7 @@ function generateFrontendPageloadPerformanceEventView(
   const conditions = new MutableSearch(searchQuery);
 
   // This is not an override condition since we want the duration to appear in the search bar as a default.
-  if (!conditions.hasFilter('transaction.duration')) {
+  if (!conditions.hasFilter('transaction.duration') && !isMetricsData) {
```

Comment:
How does metrics data work with `event.type:transaction` which is being set above? I assume it doesn't work but I'm not familiar with the metrics api.

Reply:
I'll have to check that...the Metrics API is very new and we will still have to do some updates for it to work well with performance queries

Reply:
metrics API only allows filters like `<metric-tag-name>:<metric-tag-value>…` and `event.type `is not a metric tag